src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image" src="images/imac.jpeg"
+        alt="apple"
+      >
+
+      <div class="card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="card__reit">
+        <div class="card__stars">
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
```

Comment:
where is stars block?

Reply:
> where is stars block?
at the very and 103-123

/* #region stars */
.card__stars {
  display: flex;
}

.stars_star {
  display: inline-block;
  background: url(images/star-active.svg);
  background-repeat: no-repeat;
  background-position: center;
  width: 16px;
  height: 16px;
  margin-right: 4px;
}

.stars_star:nth-child(5) {
  background: url(images/star.svg);
  background-repeat: no-repeat;
  background-position: center;
}
/* #endregion */

Reply:
I ment u have stars__start, but where is stars BEM block?

Reply:
![product card](https://user-images.githubusercontent.com/93187497/184208143-5d54c4df-a628-4fc6-8984-fe6fa4b3853f.JPG)

I`m puzzled a bit. I don't have class stars_start, only card__stars & stars_star. Both of them separated inside BEM block /* #region stars */.